package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint && npm run unit && npm run cover && npm run check-coverage && npm run snyk",
+    "test": "npm run lint && npm run unit && npm run cover && npm run check-coverage",
```

Comment:
To be clear, I think it's fine to have snyk (or similar) in the libraries just that a vulnerability on something you haven't touched shouldn't cause the tests to fail.

The solution that @easternbloc mentioned sounds a lot better.